beacon_node/lighthouse_network/src/rpc/mod.rs

```diff
@@ -151,10 +157,12 @@ pub struct NetworkParams {
 /// Implements the libp2p `NetworkBehaviour` trait and therefore manages network-level
 /// logic.
 pub struct RPC<Id: ReqId, E: EthSpec> {
-    /// Rate limiter
-    limiter: Option<RateLimiter>,
+    /// Rate limiter for our responses.
+    response_limiter: Option<ResponseLimiter<E>>,
     /// Rate limiter for our own requests.
-    self_limiter: Option<SelfRateLimiter<Id, E>>,
+    outbound_request_limiter: SelfRateLimiter<Id, E>,
```

Comment:
```suggestion
    outbound_rate_limiter: SelfRateLimiter<Id, E>,
```
Here I would follow the same naming, so that it's clear that both outbound and inbound we are rate limiting requests right?